app/static/src/app/components/WodinPlotDataSummary.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div v-if="hasData" hidden class="wodin-plot-data-summary">
+    <div class="wodin-plot-data-summary-series" v-for="(series, index) in data" :key="index"
+         :name="series.name"
+         :count="series.x?.length"
+         :x-min="Math.min(...series.x)"
+         :x-max="Math.max(...series.x)"
+         :y-min="Math.min(...series.y)"
+         :y-max="Math.max(...series.y)"
```

Comment:
If the idea is to get something easy to test, I wonder if some unique summary (e.g. a hash) might be a good idea to use?

Reply:
I guess that would be less verbose to test, but I feel like this might be more comprehensible especially when using it manually for debugging as I sometimes do! Could have both I guess - one for another ticket!